packages/core/src/awsexplorer/activation.ts

```diff
@@ -120,6 +121,20 @@ export async function activate(args: {
     for (const viewNode of viewNodes) {
         registerToolView(viewNode, args.context.extensionContext)
     }
+
+    const toolkitAuthProvider = new CommonAuthViewProvider(args.context.extensionContext, 'toolkit')
+    args.context.extensionContext.subscriptions.push(
+        vscode.window.registerWebviewViewProvider(toolkitAuthProvider.viewType, toolkitAuthProvider, {
+            webviewOptions: {
+                retainContextWhenHidden: true,
+            },
+        }),
+        vscode.commands.registerCommand('aws.explorer.setLoginService', (serviceToShow?: string) => {
```

Comment:
The purpose of this command is not clear. If it's a hacky way for a webview to call `setLoginService`, then a comment should mention that.
```suggestion
        // Hacky way for a webview to call setLoginService().
        vscode.commands.registerCommand('aws.explorer.setLoginService', (serviceToShow?: string) => {
```

Reply:
I will try to implement a non-hacky way in next commit

Reply:
>If it's a hacky way for a webview to call setLoginService, then a comment should mention that.

Command `aws.${prefix}.auth.manageConnections` is the only caller which invokes `await vscode.commands.executeCommand('aws.explorer.setLoginService', serviceToShow)`. It is not a webview.